src/util/build-functions.js

```diff
@@ -2,4 +2,11 @@ import exec from './modules/exec'
 
 export default function (PATTERN, NODE_ENV) {
   return exec('webpack', ['--bail'], { env: { PATTERN, NODE_ENV } })
+  .catch((e) => {
+    if (e.code === 'ENOENT') {
```

Comment:
execa doesn't use Bluebird for promises so predicate filtering isn't available. At least that's why I think it didn't work when I tried that.